packages/app-config-writer/src/prompt/smartlinks-config.ts

```diff
@@ -0,0 +1,173 @@
+import type { AxiosBasicCredentials } from 'axios';
+import { yellow, cyan } from 'chalk';
+import type { Choice, PromptObject } from 'prompts';
+import { prompt } from 'prompts';
+import { isAppStudio, listDestinations } from '@sap-ux/btp-utils';
+import type { ToolsLogger } from '@sap-ux/logger';
+import { FileName } from '@sap-ux/project-access';
+import { t } from '../i18n';
+import { getLocalStoredCredentials, getTargetDefinition } from '../smartlinks-config';
+import type { TargetConfig, DeployTarget } from '../types';
+
+/**
+ * @description Returns deploy questions for prompt
+ * @param target deploy target definition
+ * @returns Prompt object array of questions
+ */
+const getTargetPromptQuestions = (target?: DeployTarget) => {
+    const validator = (value: string, error: string): boolean | string => {
+        if (!value || !value.trim()) {
+            return t(error);
+        } else {
+            return true;
+        }
+    };
+    const questions: PromptObject[] = [];
+    if (isAppStudio()) {
```

Comment:
It is also possible to connect to systems via url in AppStudio. Destination should be the most common case but it is not the only.

Reply:
Hi @tobiasqueck as we are talking about a deploy "destination" and not the service source URL, I think it may be an even less common case to use blank URLs (not sure, doesn't BAS convert URLs into destinations anyway?). We could think about it in a next round, but to complete a first scope I would stick to destination support only.

Reply:
Otherwise, yes, I would be ok with just supporting destination, however, I think doing it right is actually not much more effort.

Reply:
Resolving this conversation since it is covered implictely with the other discussion about the usage of the `axios-extension`